### PR TITLE
Polish language

### PR DESCRIPTION
**Pull request by Sebek05**

### CODE REVIEW

File: data/pl.json [hunk 1]
```diff
@@ -0,0 +1,487 @@
...
+                "text": "1 000 000 ostatnich wiadomości",
+                "icon": "visibility"
+            },
+            {
+                "name": "Liczba subskrybentów, aby zobaczyć statystyki",
+                "hint": "",
+                "text": "500 i więcej",
+                "icon": "poll"
+            },
+            {
```
Comment (Sominemo) [inline data/pl.json:146]: I guess you started translating before some updates to limits were introduced
Translate and add the parts introduced in c0c7461

File: localization/languages.json [hunk 2]
```diff
@@ -6,5 +6,6 @@
     "it": ["Italian"],
     "id": ["Bahasa Indonesia"],
     "uk": "ru",
-    "be": "ru"
+    "be": "ru",
+    "pl": ["Polski"]
```
Comment (Sominemo) [inline localization/languages.json:10]: Please add Polish right after Indonesian, not at the end of the file

File: data/pl.json (hunk 1 above)
Comment (Sebek05) [reply data/pl.json:146]: Done

File: localization/languages.json (hunk 2 above)
Comment (Sebek05) [reply localization/languages.json]: Done

**Comment by Sebek05** [comment]
All Done

**Pull request merged**